hazelcast/src/main/java/com/hazelcast/config/Config.java

```diff
@@ -853,6 +855,22 @@ public JobTrackerConfig findJobTrackerConfig(String name) {
         return getJobTrackerConfig(name);
     }
 
+    public <T extends CustomServiceConfig> T getServiceConfig(String name) {
```

Comment:
Emin, I suppose you have better knowledge in Hazelcast config subsystem.
Could you please assist a little bit here ?